docker/common/add_jenkins_user.sh

```diff
@@ -18,5 +18,12 @@ chown jenkins:jenkins /var/lib/jenkins/.ccache
 # Allow writing to /usr/local (for make install)
 chown jenkins:jenkins /usr/local
 
+# Allow writing to conda root env
+if [[ "$BUILD_ENVIRONMENT" == *-conda* ]]; then
+  chown jenkins:jenkins /opt/conda
```

Comment:
Jenkins is given access only if the docker build is done by Jenkins. otherwise it works normally. We give access to jenkins to avoid corruption from any other sources.